src/applications/disability-benefits/all-claims/pages/adaptiveBenefits.js

```diff
@@ -0,0 +1,50 @@
+import {
+  benefitDescription,
+  doubleAllowanceAlert,
+} from '../content/adaptiveBenefits';
+
+export const uiSchema = {
+  'ui:title': 'Automobile allowance and adaptive benefits',
+  'ui:description': benefitDescription,
+  'view:modifyingHome': {
+    'ui:title': 'Do you need help buying or modifying your home?',
+    'ui:widget': 'yesNo',
+  },
+  'view:modifyingCar': {
+    'ui:title': 'Do you need help buying or modifying your car?',
+    'ui:widget': 'yesNo',
+  },
+  'view:alreadyClaimedVehicleAllowance': {
+    'ui:title': 'Have you ever been granted an automobile allowance?',
+    'ui:widget': 'yesNo',
+    'ui:options': {
+      expandUnder: 'view:modifyingCar',
+    },
+  },
+  'view:doubleAllowanceAlert': {
+    'ui:description': doubleAllowanceAlert,
+    'ui:options': {
+      expandUnder: 'view:modifyingCar',
+      hideIf: formData => !formData['view:alreadyClaimedVehicleAllowance'],
```

Comment:
I think it'll work either way, but might want to access this property with `_.get()` and set a default value for clarity. But 🤷‍♂️

Reply:
:man_shrugging:
This will only error out if `formData` is `undefined`, but I think at that point, we have other issues. :stuck_out_tongue:

Reply:
After nesting, it was a couple properties deep, so I switched to `get` anyway. :man_shrugging: 

All the :man_shrugging:s!